spec/requests/device_story_comments_controller_spec.rb

```diff
@@ -30,7 +30,6 @@
 
       expect { post '/device_stories/1/device_story_comments', params: { device_story_comment: comment_params } }.to change { DeviceStoryComment.count }.by(0)
       expect(response.status).to eq 302
-      expect(WebMock).not_to have_requested(:post, addressable)
```

Comment:
@matschaffer This is related to the `.save` coming before `.spam?` bug. Fixing that caused Rspec to fail the test because now there were 2 requests coming through. As I understand it this line expects no requests to have made to Akismet but that did not make sense for me since requests must be made to check for spam.